packages/service-library/tests/deferred_tasks/test__base_deferred_handler.py

```diff
@@ -445,7 +445,7 @@ async def _run_that_times_out(_: DeferredContext) -> None:
 
     await _assert_mock_call(mocks, key=MockKeys.ON_FINISHED_WITH_ERROR, count=1)
     for entry in mocks[MockKeys.ON_FINISHED_WITH_ERROR].call_args_list:
-        assert "asyncio.exceptions.TimeoutError" in entry.args[0].error
+        assert "builtins.TimeoutError" in entry.args[0].error
```

Comment:
See: https://docs.python.org/3.11/library/asyncio-task.html#asyncio.wait_for

> Changed in version 3.11: Raises [TimeoutError](https://docs.python.org/3.11/library/exceptions.html#TimeoutError) instead of [asyncio.TimeoutError](https://docs.python.org/3.11/library/asyncio-exceptions.html#asyncio.TimeoutError).